cli/src/api/mod.rs

```diff
@@ -110,7 +123,14 @@ impl PhylumApi {
         let body = response.text().await?;
 
         if !status_code.is_success() {
-            let err = ResponseError { body, code: status_code };
+            let details = if let Ok(err) = serde_json::from_str::<ApiJsonError>(&body) {
+                log::debug!("Error ID: {}", err.error.error_id);
+                err.error.description
+            } else {
+                body
+            };
+            let err = ResponseError { details, code: status_code };
```

Comment:
Since you're re-wrapping the error here, I'd assume the previous work to have status-code based error message overrides still works?

Reply:
That's correct:

```
❯ phylum project create test
❗ Project 'test' already exists
```